src/common/darktable.h

```diff
@@ -39,7 +39,7 @@
 #define O_BINARY 0
 #endif
 
-#include "ThreadSafetyAnalysis.h"
+#include "external/ThreadSafetyAnalysis.h"
```

Comment:
I think this should be split out and put into a separate patch?

Reply:
did so